src/Storefront/Resources/views/storefront/page/account/profile/recover-password.html.twig

```diff
@@ -9,8 +9,11 @@
                         <form method="post"
                               class="account-recover-password-form card"
                               action="{{ path('frontend.account.recover.request') }}"
-                              data-form-validation="true"
-                              data-form-submit-loader="true">
+                              {# @deprecated tag:v6.7.0 - Form validation plugin will be replaced by form handler plugin. #}
+                              {% if feature('ACCESSIBILITY_TWEAKS') %}data-form-handler="true"{% endif %}
```

Comment:
Could we also do this as if-else instead of multiple checks?

```twig
<form
  action="{{ path('frontend.account.recover.request') }}"
  {# @deprecated tag:v6.7.0 - Form validation plugin will be replaced by form handler plugin. #}
  {% if feature('ACCESSIBILITY_TWEAKS') %}
    data-form-handler="true"
  {% else %}
    data-form-validation="true"
    data-form-submit-loader="true"
  {% endif %}>
```